Readme.md

```diff
@@ -21,7 +21,7 @@ You get:
 
 ### Server
 
-`deploy to heroku button here`
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
```

Comment:
Perhaps use the `.svg` button instead?
`https://www.herokucdn.com/deploy/button.svg`
![](https://www.herokucdn.com/deploy/button.svg)

----

For those wondering about the missing `template` parameter in the deploy URL see: [*Using an implicit template*](https://devcenter.heroku.com/articles/heroku-button#using-an-implicit-template)